ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -248,50 +252,52 @@ define([
 
     repositionLatLon: function() {
       if (this.get('usngbb') !== undefined) {
-        var result = converter.USNGtoLL(this.get('usngbb'))
-        var newResult = {}
-        newResult.mapNorth = result.north
-        newResult.mapSouth = result.south
-        newResult.mapEast = result.east
-        newResult.mapWest = result.west
-
-        this.set(newResult)
+        try {
+          var result = converter.USNGtoLL(this.get('usngbb'))
+          var newResult = {}
+          newResult.mapNorth = result.north
+          newResult.mapSouth = result.south
+          newResult.mapEast = result.east
+          newResult.mapWest = result.west
+
+          this.set(newResult)
+        } catch (err) {}
```

Comment:
❓ this will result in a silent failure since this error is just swallowed here. It would at least be helpful to log it

Reply:
logging the error may result in a cross-origin error that destroys the anyGeo component

Reply:
https://github.com/codice/usng.js/issues/21 needs to be resolved before I can remove this try catch